WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -52,6 +53,8 @@ enum FeatureFlag: Int, CaseIterable {
             return BuildConfiguration.current != .appStore
         case .sidebar:
             return false
+        case .serif:
+            return BuildConfiguration.current != .appStore
```

Comment:
I added a flag to switch to serif fonts in jpios for testing - adding some WP flare.